src/olympia/reviewers/models.py

```diff
@@ -1062,8 +1062,15 @@ def check_uses_native_messaging(cls, version):
 
     @classmethod
     def check_is_locked(cls, version):
-        locked = get_reviewing_cache(version.addon.pk)
-        return bool(locked) and locked != settings.TASK_USER_ID
+        # Langpacks are submitted as part of Firefox release process and should
+        # never be blocked by a reviewer having opened the review page - they
+        # should always be auto-approved anyway.
+        is_langpack = version.addon.type == amo.ADDON_LPAPP
+        locked_by = get_reviewing_cache(version.addon.pk)
+        return (
+            not is_langpack and
+            bool(locked_by) and
```

Comment:
nit: isn't `locked_by` always an int (or null)?  Why do we need to force it to `bool`?

Reply:
It's to force the method to always return a boolean. Not stricly necessary but makes it more predictable / easier to use & test.